linux_os/guide/system/network/network_implement_access_control/tests/correct_tcpwrappers_allow_config.pass.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "192.168.122.25" >> /etc/hosts.allow
```

Comment:
I think you should ensure that the firewalld is disabled.

Reply:
Both comments addressed